astropy/timeseries/periodograms/lombscargle/implementations/scipy_impl.py

```diff
@@ -12,10 +12,10 @@ def lombscargle_scipy(t, y, frequency, normalization='standard',
 
     Parameters
     ----------
-    t, y: array_like  (NOT astropy.Quantities)
```

Comment:
move `NOT astropy.Quantities` to text (or just remove?)

Reply:
Moved ✅